src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -160,7 +160,7 @@ public function setup(): void
         $this->exchange()->declareExchange();
 
         $this->queue()->declareQueue();
-        $this->queue()->bind($this->exchange()->getName(), $this->queueConfiguration['routing_key'] ?? null);
+        $this->queue()->bind($this->exchange()->getName(), $this->queueConfiguration['routing_key'] ?? null, $this->queueConfiguration['arguments'] ?? null);
```

Comment:
Shouldn't we name this `bind_arguments` instead of just `arguments`?

Reply:
Yes I think it would be a better idea

Reply:
I will make the changes in this direction. You have to modify the messenger configuration of the framework bundle to support this options

Reply:
> Shouldn't we name this `bind_arguments` instead of just `arguments`?

Hi
You have not considered my request since your comments were taken into account.